source/WeaponSpecial/Swift_test.gd

```diff
@@ -0,0 +1,10 @@
+extends "res://source/unit/WeaponSpecial.gd"
+class_name Swift_test
```

Comment:
I'd remove this class name for now.
We don't really want this to be a class and it's cluttering the API.